fmriprep/cli/parser.py

```diff
@@ -533,9 +533,16 @@ def _bids_filter(value):
         "improve FMRIPREP and provides an indicator of real "
         "world usage crucial for obtaining funding.",
     )
+    g_other.add_argument(
+        "--debug",
+        action="store",
+        nargs="+",
+        choices=config.DEBUG_MODES + ("all",),
+        help="Debug mode(s) to enable. 'all' is alias for all available modes.",
+    )
+
     g_other.add_argument(
         "--sloppy",
-        dest="debug",
```

Comment:
dropping this will require renaming of current `debug` variables in the workflow:

https://github.com/poldracklab/fmriprep/blob/d7fc498b4d58a60ec174d0ec088770bdfed5483f/fmriprep/workflows/base.py#L240
https://github.com/poldracklab/fmriprep/blob/3ca515c4a63966480c0a93c0b36192e19d3888dd/fmriprep/workflows/bold/base.py#L336

Reply:
Ah, good catch.